addons/stock/report/report_deliveryslip.xml

```diff
@@ -119,7 +127,21 @@
                                 <!-- If not printing lots/serial numbers => merge lines with same product -->
                                 <t t-else="" name="aggregated_move_lines">
                                     <t t-set="aggregated_lines" t-value="o.move_line_ids._get_aggregated_product_quantities()"/>
-                                        <t t-call="stock.stock_report_delivery_aggregated_move_lines"/>
+                                    <t t-call="stock.stock_report_delivery_aggregated_move_lines"/>
+                                    <t t-set="empty_lines" t-value="o.move_lines.filtered(lambda x: x.product_uom_qty and x.quantity_done == 0)"/>
+                                    <tr t-foreach="empty_lines" t-as="move">
+                                        <td>
+                                            <span t-field="move.product_id"/>
+                                            <p t-if="move.description_picking != move.product_id.name">
+                                                <span t-field="move.description_picking"/>
+                                            </p>
+                                        </td>
+                                        <td class="text-center">
+                                            <span t-field="move.product_uom_qty"/>
+                                            <span t-field="move.product_uom"/>
+                                        </td>
+                                        <td/>
```

Comment:
Do you think we should print 0.00 _uom_ instead? To keep the same logic when we have a reserved move line without qty_done ?

Reply:
Nope, it's a part of the spec. But there is nothing about the fact we can have *0.00 uom* when the transfer is not done yet, but I think it's a big deal.